include/pokemon_3.h

```diff
@@ -1,10 +1,30 @@
 #ifndef GUARD_POKEMON_3_H
 #define GUARD_POKEMON_3_H
 
+struct EvolveStatus
+{
+    u8 evoItem1;
+    u8 evoItem2;
+    u8 unk6;
+    u16 evolutionConditionStatus;
+    s16 targetEvolveSpecies;
+};
+
+bool8 HasRecruitedMon(s16 species_);
```

Comment:
Is the trailing underscore needed?

Reply:
Yes, to avoid shadowing the cast to `s32`. I can change it to `_species` if you prefer. Or flip it to `species` for the arg and do `species_s32` for the cast.

```c
bool8 HasRecruitedMon(s16 species_) {
    s32 species = species_;
    ...
```

Reply:
I like the latter suggestion of swapping the names so that the parameter is a clean `species`. The function prototype is the public interface, so it's better to have clean names there.